Libraries/StyleSheet/StyleSheetValidation.js

```diff
@@ -15,6 +15,7 @@ var ImageStylePropTypes = require('ImageStylePropTypes');
 var ReactPropTypeLocations = require('ReactPropTypeLocations');
 var TextStylePropTypes = require('TextStylePropTypes');
 var ViewStylePropTypes = require('ViewStylePropTypes');
+var TextInputStylePropTypes = require('TextInputStylePropTypes');
```

Comment:
Actually, I don't think this should be in the core style stuff anyway - it should be a separate prop since it's so specific to TextInput.